test/cli_test.py

```diff
@@ -395,6 +399,24 @@ def touch_art(item: Item, image_path: Path):
         item = album.items().get()
         assert_has_embedded_artwork(self.get_path(item), self.IMAGE_FIXTURE2)
 
+        # now set a maxwidth and verify the final image has the right
+        # dimensions
+        touch_art(item, image_path)
+        self.config["alternatives"]["myexternal"]["album_art_maxwidth"] = 1
+        self.runcli("alt", "update", "myexternal")
+        mediafile = MediaFile(self.get_path(item))
+        tmp_embedded_image = tmp_path / "embedded_image.png"
+        with open(tmp_embedded_image, "wb") as binary_file:
+            binary_file.write(mediafile.art)
+        art_resizer = ArtResizer()
+        (width, height) = art_resizer.get_size(bytes(tmp_embedded_image))
```

Comment:
Ruff and pyright complain here (`poetry run ruff check && poetry run pyright`). You can use `# pyright: ignore`.